### ISSUE TITLE
Barnes and Noble doesn't work

### ISSUE THREAD

**Issue by hellomiakoda** [apprenticeharper/DeDRM_tools#1641]
Attempting to get new Barnes and Noble key just insists the password is wrong/can't connect, even though I went and tried the password on the site and it's the right password.

**Comment by ElleKayEm** [comment]
Yes, the method of retrieving the key in DeDRM broke when BN cut off the old desktop Nook apps.  See discussion in issue #814.  A write up on how to extract the key from a virtual Android Nook app is here:  https://www.aricrenzo.com/2019-12-13-Liberate-Your-Nook-Ebooks/
